include/openspace/rendering/loadingscreen.h

```diff
@@ -49,24 +50,19 @@ class LoadingScreen {
 public:
     BooleanType(ShowMessage);
     BooleanType(ShowNodeNames);
-    BooleanType(ShowProgressbar);
     BooleanType(CatastrophicError);
 
-    LoadingScreen(ShowMessage showMessage, ShowNodeNames showNodeNames,
-        ShowProgressbar showProgressbar);
+    LoadingScreen(ShowMessage showMessage, ShowNodeNames showNodeNames);
     ~LoadingScreen();
 
     void render();
+    void renderLogMessages() const;
```

Comment:
You can move this function down to the `private` section since noone else should be calling this anyway